tests/dat/actions/wcbin.js

```diff
@@ -5,25 +5,26 @@
 function main(params) {
     var str = params.payload;
     console.log("The payload is '" + str + "'");
-    whisk.invoke({
-        name : 'wc',
-        parameters : {
-            payload : str
-        },
-        blocking : true,
-        next : function(error, activation) {
+
+    var promise = new Promise(function(resolve, reject) {
+        whisk.invoke({
```

Comment:
I like it and actually started with this exact same syntax:)  Started second guessing myself though and changed it.  Will test it out and go with this if it works.